staging/src/k8s.io/apiserver/pkg/server/config.go

```diff
@@ -527,6 +527,8 @@ func (c completedConfig) New(name string, delegationTarget DelegationTarget) (*G
 
 	s.listedPathProvider = routes.ListedPathProviders{s.listedPathProvider, delegationTarget}
 
+	s.openAPIGetter = apiopenapi.NewSchemaCacher(s.UnprotectedHandler(), "/openapi/v2")
```

Comment:
Does it seem appropriate to hard code in the path "/openapi/v2" here? We have it hard coded in [server/routes/openapi.go](https://github.com/jennybuckley/kubernetes/blob/c6c5369dd06a789583ce526a4075a00fdc6cce37/staging/src/k8s.io/apiserver/pkg/server/routes/openapi.go#L42) as well which I think makes sense and I'm pretty sure we can rely on this endpoint continuing to be served for the foreseeable future.

Reply:
Good point, that might break when we want to go for openapi v3, which I'm sure is going to happen.

Reply:
I think the least we could do is make the path a const in server/routes/openapi.go and then have both of the values point to it

Reply:
yes